src/ResourceManager/Network/Commands.Network/ServiceEndpointPolicy/ServiceEndpointPolicyDefinition/GetAzureServiceEndpointPolicyDefinitionCommand.cs

```diff
@@ -34,23 +34,10 @@ public class GetAzureServiceEndpointPolicyDefinitionCommand : NetworkBaseCmdlet
             HelpMessage = "The Service endpoint policy")]
         public PSServiceEndpointPolicy ServiceEndpointPolicy { get; set; }
 
-        [Parameter(
-            Mandatory = true,
```

Comment:
This is a breaking change (removing amandatory param)... why are you pulling out the get by resourceID?

Reply:
this was a bug. This should not be there in first place. There is no server call to query with resourceId. We cannot have this sort of a query for this API.

Reply:
@chandrasekarsrinivasan 
The Cmdlet is not using the resourceId anyways, 
It is [pulling out the name from the resourceId](https://github.com/Azure/azure-powershell/blob/f6a2882e0aa408b44465aca4226021547fa31be7/src/ResourceManager/Network/Commands.Network/ServiceEndpointPolicy/ServiceEndpointPolicyDefinition/GetAzureServiceEndpointPolicyDefinitionCommand.cs#L51) and using that to [make the call](https://github.com/Azure/azure-powershell/blob/f6a2882e0aa408b44465aca4226021547fa31be7/src/ResourceManager/Network/Commands.Network/ServiceEndpointPolicy/ServiceEndpointPolicyDefinition/GetAzureServiceEndpointPolicyDefinitionCommand.cs#L62 into the service. 

Can you elaborate on what the bug is?